### ISSUE TITLE
[Bug]: Fractional Scaling Breaks swww

### ISSUE THREAD

**Issue by SizzinSeal** [prasanthrangan/hyprdots#1317]
### Description

Wallpaper breaks when hyprland scaling is set to a fraction

### Steps to Reproduce

1. set monitor scaling to a fraction, e.g `monitor = eDP-1,1920x1080@300,0x0,1.25` in hyprland.conf

### Expected Behavior

Wallpaper should fit to screen

### Actual Behavior

Wallpaper does not fit to screen

### Screenshots

![image](https://github.com/prasanthrangan/hyprdots/assets/111480281/66c3667c-92f6-4880-a480-7a4c6492028e)

### Environment

- Linux Distro: CachyOS (Arch based)
- Linux Version: 6.8.6-2-cachyos:
- Hyprland Version: v0.38.1
- Installation Environment: standalone (no dual boot)
- Display resolution: 1920x1080@300, ~15inch so HiDPI

### Additional Information

swww query yield the following result:
```
eDP-1: 960x540, scale: 2, currently displaying: color: 000000
```

this is a brand new install of the OS, with no other packages installed. Setting scale to 1, or 2, and it works properly

**Comment by hdm9527** [comment]
https://github.com/LGFae/swww/issues/279

**Comment by kRHYME7** [comment]
Please update, swww upstream does acknowledge this issue and already fix on their latest release Thanks

**Comment by itsmenewbie03** [comment]
I have latest swww 
![image](https://github.com/prasanthrangan/hyprdots/assets/89230263/f206872b-f1c2-4dd8-ab2a-b4d66f528fa2)
But still the wallpaper is not fitting the screen
![image](https://github.com/prasanthrangan/hyprdots/assets/89230263/b98b21b1-7b0e-4170-b29d-ec19d0c55c6e)

**Comment by kRHYME7** [comment]
https://github.com/LGFae/swww/releases/tag/v0.9.3

Hyde repo can't actually resolve this issue, so I closed it. 

For now use swww-git ``` yay -Sy swwww-git ``` 


https://aur.archlinux.org/packages/swww

or be sure to have swww 0.9.3 then reboot.

**Comment by SizzinSeal** [comment]
I had 0.9.3, and I just tried swww-git. The issue persists on both of them

**Comment by kRHYME7** [comment]
Got it! I will reopen this issue so that we got no dupes. And will report in the upstream

**Comment by kRHYME7** [comment]
To downgrade:
Download this PKGBUILD
https://aur.archlinux.org/cgit/aur.git/commit/?h=swww&id=a8247fc4d874fb4b6a99b946d6a40cbeef29088a

Extract and cd to the extracted directory. 

run ``` makepkg -si ```  Will compile swww 0.9.1

**Comment by kRHYME7** [comment]
swww-git now has version 10 ig but compiling it is too much for my laptop 😭  (and dunno if it has been fixed yet at v0.10 )

**Comment by SizzinSeal** [comment]
swww recently release v0.9.4, which fixes this problem. Its not yet on the `swww` AUR package but it is on the `swww-git` package. Installing `swww-git` installs v0.9.4 and fixes the problem
